packages/store-ui/src/Carousel/ArrowLeft.tsx

```diff
@@ -1,6 +1,7 @@
-import type { ComponentProps, FC } from 'react'
 import React from 'react'
-import { SliderArrowLeft } from '@vtex/store-ui'
+import type { ComponentProps, FC } from 'react'
+
+import { SliderArrowLeft } from '../index'
```

Comment:
There is just one thing that I'm not really sure. If we import this way, do we perform tree shaking + esm import correctly ? If not, we should import the component by its full path to avoid including the whole bundle of store-ui.
```suggestion
import { SliderArrowLeft } from '../Slider/ArrowLeft'
```

Reply:
Good point! But I think we're performing there. How can I confirm? @tlgimenes